lib/real_world/user/queries/index_users_by_username.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'colmena/query'
+
+module RealWorld
+  module User
+    module Queries
+      class IndexUsersByUsername < Colmena::Query
+        def call(usernames:)
+          users = port(:repository).index_by_usernames(usernames)
+
+          response(users)
+        end
+      end
+    end
+  end
+end
```

Comment:
There's a spec missing here

Reply:
And if you use `response`, I don't understand how this is possible

https://github.com/schoolhouse-io/colmena-realworld-example-app/pull/29/files/07e7a0a370b8bc663fecf34b8dae3a6917dcb10c#diff-c1cefe2adb8130ccde10f4ebba77c926R26

Reply:
Looks like a bug to me.